cluster/addons/dns/kube2sky/Changelog

```diff
@@ -1,3 +1,9 @@
+## Version 1.15 (Apr 7 2016 Lucas Käldström <lucas.kaldstrom@hotmail.co.uk>)
+ - No code changes since 1.14
```

Comment:
I don't know which version `1.14` was built with (as it was using go on host), but adding a note about go1.6